datalad_next/runners/iter_subproc.py

```diff
@@ -1,122 +0,0 @@
-from __future__ import annotations
-
-from pathlib import Path
-from typing import (
-    Iterable,
-    List,
-)
-from datasalad.runners import CommandError as SaladCommandError
-from datasalad.iterable_subprocess import iterable_subprocess
-
-from datalad_next.exceptions import CommandError
-from datalad_next.consts import COPY_BUFSIZE
-
-__all__ = ['iter_subproc']
-
-
-def iter_subproc(
-    args: List[str],
-    *,
-    input: Iterable[bytes] | None = None,
-    chunk_size: int = COPY_BUFSIZE,
-    cwd: Path | None = None,
-    bufsize: int = -1,
-):
```

Comment:
There are a few test failures in this PR and also in #783:

```
E       TypeError: iter_subproc() got an unexpected keyword argument 'input'
```

I tried to dig into them a bit, and found https://github.com/datalad/datasalad/commit/4b448912ded25e91f9a6c763620d921cb894a8c0, where the "input" parameter was renamed to "inputs". 

I will try to push a change that adjusts to the different parameter name, but I suspect that the problem is that different runners either expect "input" or "inputs"? Let's see...